DLCS.Repository/Messaging/MessageBus.cs

```diff
@@ -0,0 +1,277 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using DLCS.Core.Settings;
+using DLCS.Model.Assets;
+using DLCS.Model.Messaging;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+namespace DLCS.Repository.Messaging
+{
+    public class MessageBus : IMessageBus
```

Comment:
Reading the name of this and seeing impl feels at odds. I'd expect `MessageBus` to be putting stuff onto a queue rather than calling direct. 

Would this be better in the API project? Would anywhere else be likely to use it?

And/or would `EngineMessage<Something>` be a better name?

Reply:
At the moment the only implemented thing on this class is the direct engine call, which is not the standard sort of message - but won't it also later be enqueueing  incoming assets from API calls to the queue, broadcasting asset change events etc?